src/CSET/operators/filters.py

```diff
@@ -21,30 +21,30 @@
 
 
 def filter_cubes(
-    cubelist: iris.cube.CubeList, constraint: iris.Constraint, **kwargs
+    cube: iris.cube.Cube, constraint: iris.Constraint, **kwargs
 ) -> iris.cube.Cube:
     """
     Filters a cubelist down to a single cube based on a constraint.
 
     Arguments
     ---------
-    cubelist: iris.cube.CubeList
-        Cubes to iterate over
+    cube: iris.cube.Cube | iris.cube.CubeList
+        Cube(s) to iterate over
     constraint: iris.Constraint
         Constraint to extract
 
     Returns
     -------
-    cube: iris.cube.Cube
-        Single variable
+    filtered_cube: iris.cube.Cube
```

Comment:
```suggestion
    iris.cube.Cube
```
Don't need to name the output variable, just give its type.